dataprofiler/tests/data_readers/test_csv_data.py

```diff
@@ -91,6 +91,11 @@ def setUpClass(cls):
             dict(path=os.path.join(test_dir, 'csv/quote-test.txt'),
                  count=10, delimiter=' ', has_header=[0, None],
                  num_columns=3, encoding='utf-8'),
+            
+            #dict(path=os.path.join(test_dir, 'csv/quote-test-singlequote.txt'),
+            #     count=9, delimiter=' ', has_header=[0, None],
+            #     num_columns=3, encoding='utf-8'),
```

Comment:
can you remove this?

Reply:
Left to denote an issue, will need to remove next PR. Currently, there's an issue *reloading* (not the initial load) for a file with a single quote.